actionview/lib/action_view/helpers/javascript_helper.rb

```diff
@@ -72,6 +79,10 @@ def javascript_tag(content_or_options_with_block = nil, html_options = {}, &bloc
             content_or_options_with_block
           end
 
+        if html_options[:nonce] == true
```

Comment:
Might be worth defaulting this to true? Seems pretty fair to default to trusting any JS the server is deliberately sending.

Reply:
Possibly - though the thought of defaulting anything to less secure makes me nervous 😟